loader/src/main/java/net/neoforged/fml/ModLoader.java

```diff
@@ -117,7 +117,7 @@ public static void gatherAndInitializeMods(final Executor syncExecutor, final Ex
         if (!failedBounds.isEmpty()) {
             LOGGER.fatal(CORE, "Failed to validate feature bounds for mods: {}", failedBounds);
             for (var fb : failedBounds) {
-                loadingIssues.add(ModLoadingIssue.error("fml.modloading.feature.missing", null, fb, ForgeFeature.featureValue(fb)).withAffectedMod(fb.modInfo()));
```

Comment:
If you remove the `null` here you need to shift many indices in the translations. That's why it's still there (previously this was the "mod loading stage" which I removed; but I didn't want to adjust every single lang entry).